setup.py

```diff
@@ -50,20 +50,21 @@ def run(cls):
     license="Apache License 2.0",
     description="Pure Python client for Apache Kafka",
     long_description=README,
-    keywords="apache kafka",
+    keywords=[
+        "apache kafka",
+        "kafka",
+    ],
     classifiers=[
         "Development Status :: 5 - Production/Stable",
         "Intended Audience :: Developers",
         "License :: OSI Approved :: Apache Software License",
         "Programming Language :: Python",
-        "Programming Language :: Python :: 2",
```

Comment:
why drop the python2 support notice?

Reply:
If we're dropping support, we should probably add `python_requires` to the setup.py file.

Reply:
I'm 👍 on dropping python 2 support (in fact I'm all for aligning with upstream Python EOL cadence, as that seems pretty conservative to have 5 years to support a version), but I'd do it separate from this PR... keep this one focused on the CI workflow migration.

Reply:
Sounds good, I'll revert my tag drop for now.